src/App.jsx

```diff
@@ -13,6 +14,7 @@ function App() {
       <NavBar />
       <Routes>
         <Route path="/api/reviews" element={<GetReviews />} />
+        <Route path="/api/reviews/:review_id" element={<SingleReview />} />
```

Comment:
These url's are separate from your backend api endpoints, these are strictly for navigating your front end site so they should not have api at the front of them